src/foam/java/Class.js

```diff
@@ -19,12 +19,26 @@ foam.CLASS({
   package: 'foam.java',
   name: 'Class',
 
+  constants: [
+    {
+      name: 'SUPER_CLASSES',
```

Comment:
documentation: 'Store Java Class references so they can be accessed by subclasses.'